### PR TITLE
Initial devenv setup

### PR DESCRIPTION
**Pull request by Lugarun**

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source_url "https://raw.githubusercontent.com/cachix/devenv/95f329d49a8a5289d31e0982652f7058a189bfca/direnvrc" "sha256-d+8cBpDfDBj41inrADaJt+bDWhOktwslgoP5YiGJ1v0="
+
+use devenv
```
Comment (mosiman) [inline .envrc:3]: no newline? is this even a file?

File: db/db.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+                                  elo FLOAT,
+                                  botfile VARCHAR,
+                                  botstatus VARCHAR);
+
+CREATE TABLE IF NOT EXISTS games (gameid SERIAL PRIMARY KEY,
+                                  user1 INTEGER REFERENCES users(userid),
+                                  user2 INTEGER REFERENCES users(userid),
+                                  winner INTEGER REFERENCES users(userid),
+                                  generated_at TIMESTAMP,
+                                  gamefile VARCHAR)
```
Comment (mosiman) [inline db/db.sql:12]: if this is for testing stuff, maybe create the tables in it's own schema so it doesn't interfere with api / other stuff?

```
create schema if not exists rps_test;
create table if not exists rps_test.users ( ... );
...
```

**Review by mosiman — APPROVED** [review]

File: .envrc (hunk 1 above)
Comment (Lugarun) [reply .envrc:3]: This integrates devenv with https://direnv.net/ so if you have direnv installed you will automatically enter the devenv shell when you navigate to this project's root directory.

Comment (Lugarun) [reply .envrc:3]: I'll fix the line ending

File: db/db.sql (hunk 2 above)
Comment (Lugarun) [reply db/db.sql:12]: Discussed offline: I am keen on staying in sync with your database schema.

**Review by japorized — APPROVED** [review]

**Review by mosiman — APPROVED** [review]

**Pull request merged**